keel/Core/storage_backends.py

```diff
@@ -14,3 +15,8 @@ class PrivateMediaStorage(S3Boto3Storage):
     file_overwrite = False
     custom_domain = False
 
+doc_storage = None
+if settings.DEBUG == True:
```

Comment:
instead we can use the settings module only
either production or development

Reply:
can you specify which variable?

Reply:
DJANGO_SETTINGS_MODULE
Debug does not accurately defines what platform we are using
one can perform DEBUG on staging env or any other env also....which clearly breaks this scenario then